Formula/etcd.rb

```diff
@@ -42,6 +59,7 @@ def plist
           <key>Label</key>
           <string>#{plist_name}</string>
           <key>ProgramArguments</key>
+          #{plist_unsupported_platformenv}
```

Comment:
Doesn't this break the `plist`, since the array [*] below is now a value with no key?

[*] This one:
```
<array>
  <string>#{opt_bin}/etcd</string>
</array>
```
I think this also sets the environment variable for an entirely separate process, which doesn't seem to be what's needed here.

The correct entry to add to the `plist` is probably (from `man launchd.plist`):
```
     EnvironmentVariables <dictionary of strings>
     This optional key is used to specify additional environmental variables to be set before running the job. Each
     key in the dictionary is the name of an environment variable, with the corresponding value being a string repre-
     senting the desired value.  NOTE: Values other than strings will be ignored.
```

Reply:
I've not used plists much.
I'll fix it to add that instead.